doc/run_experiment.rst

```diff
@@ -920,7 +922,7 @@ Output files
 
 The result, log, model, and prediction files generated by run_experiment will
 all share the following automatically generated prefix
```

Comment:
Might as well drop "following" here too. I think I had it originally before I added the section that broke down what each part meant on separate lines.